exercises/practice/equilateral_triangle/equilateral_triangle_test.py

```diff
@@ -0,0 +1,131 @@
+import unittest
+from z3 import *
+from equilateral_triangle import *
+
+class EquilateralTriangleTest(unittest.TestCase):
+    NUM_DECIMAL_PLACES = 5
+
+    def setUp(self):
+        # Display rational numbers as decimals instead of fractions
+        set_option(rational_to_decimal=True)
+
+        # Set Precision to 5 decimal places of outputted rational numbers
+        set_option(precision=EquilateralTriangleTest.NUM_DECIMAL_PLACES)
+
+    def test_same_y(self):
+        first_point = (0, 0)
```

Comment:
This is a lot of repeated code. Might be easier to package most inputs and expect outputs into a list, write a check function, then go through everything in a for loop. See https://github.com/grantg012/exercism-z3/blob/goldbach/exercises/practice/goldbach/goldbach_test.py

Reply:
I agree that there is a lot of repeated code and it should be condensed.  However, I do not think having one test case method is not such a good idea because if that test case method were to fail, you would not know which of the many inputs failed on that test case (unless there is some verbose detailing of the tests that I am unaware about).  I will soon implement a check function, and will combine the non-edge cases into one test case method.